pkg/server/operands_handler.go

```diff
@@ -0,0 +1,100 @@
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"os/user"
+	"path/filepath"
+
+	"github.com/openshift/console/pkg/serverutils"
+	"github.com/operator-framework/kubectl-operator/pkg/action"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GetConfig() (*rest.Config, error) {
+	// If an env variable is specified with the config location, use that
+	if len(os.Getenv("KUBECONFIG")) > 0 {
+		return clientcmd.BuildConfigFromFlags("", os.Getenv("KUBECONFIG"))
+	}
+	// If no explicit location, try the in-cluster config
+	if c, err := rest.InClusterConfig(); err == nil {
+		return c, nil
+	}
+	// If no in-cluster config, try the default location in the user's home directory
+	if usr, err := user.Current(); err == nil {
+		if c, err := clientcmd.BuildConfigFromFlags(
+			"", filepath.Join(usr.HomeDir, ".kube", "config")); err == nil {
+			return c, nil
+		}
+	}
+
+	return nil, fmt.Errorf("could not locate a kubeconfig")
+}
+
+func operandsListHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
+		w.Header().Set("Allow", "GET")
+		serverutils.SendResponse(w, http.StatusMethodNotAllowed, serverutils.ApiError{Err: "Method unsupported, the only supported methods is GET"})
+		return
+	}
+	query := r.URL.Query()
+	operatorName := query.Get("name")
+	operatorNamespace := query.Get("namespace")
+	if len(operatorName) < 1 || len(operatorNamespace) < 1 {
```

Comment:
Should there be a `return` at the end of this if block (and the other error handling blocks) so that we don't execute the rest of the logic in this func? I assume once we write the error response, we don't need to continue execution.

Reply:
Thanks @TheRealJon 👍  nice catch